tensorflow/c/experimental/filesystem/modular_filesystem_test.cc

```diff
@@ -1569,7 +1590,7 @@ TEST_P(ModularFileSystemTest, TestRoundTrip) {
   if (!status.ok())
     GTEST_SKIP() << "NewRandomAccessFile() not supported: " << status;
 
-  char scratch[64 /* big enough to accommodate test_data */] = {0};
+  char scratch[64] = {0};
```

Comment:
We should keep the comments to ensure future changes of the test don't create a buffer that is too small

Reply:
I have to comment this test when it passed because the test time is very long. So I have to remove this comment. It should not be removed when the PR is merge

Reply:
Oh, now this makes sense.

I usually comment with `#if 0` before the block and `#endif` after. These can also be nested, unlike `/* ... */` comments. Plus, I can easily replace with `#if DEBUG_TEST_CONDITION_X` instead of `0` and when I compile I pass in `-DDEBUG_TEST_CONDITION_X=0` (or 1) to conditionally enable/disable these blocks